nilearn/plotting/surf_plotting.py

```diff
@@ -533,7 +569,8 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
               hemi='left', view='lateral', engine='matplotlib',
               cmap=None, symmetric_cmap=False, colorbar=False,
               avg_method='mean', threshold=None, alpha='auto',
-              bg_on_data=False, darkness=1, vmin=None, vmax=None,
+              bg_on_data=False, bg_map_rescale="auto", darkness=.5,
```

Comment:
I think we should keep `darkness=1.` as the default.

plot with the default darkness in the `main` branch for the sulcal depth:

![fig1](https://user-images.githubusercontent.com/9196501/210331949-cba9f734-9b5c-448a-8499-8f21addd448d.png)

and with this PR's default of darkness=.5:

![fig2](https://user-images.githubusercontent.com/9196501/210332002-9dcfe9a2-14f0-4b0d-97d9-ea67cedd41df.png)


so if we make that change existing code that uses the sulcal depth (as it was the only bg map available so far) and doesn't specify a value for `darkness` will start creating worse plots than before

Reply:
I changed the default value for `darkness` because I thought the second plot looked nicer.
I like that it has higher contrasts for pieces of information users are interested in (probability maps), which I think is an interesting feature when such figures are seen in black and white (which often happens if you print research articles).

But I obviously don't want to push this too hard and would happily change the default back to 1.
Maybe .7 or .8 would actually be a good compromise (again, I don't want to seem to be pushy about it haha).

Reply:
ok, if it's an intentional choice fair enough! I'd still push for a value a bit closer to 1 with the current choice I find it hard to see anything in the background map

Reply:
Great I changed it to `.7` so that background maps' contrast is lower than that of probability maps, but still more visible than they are when `darkness=.5`.

For comparison, here are two screenshots with `bg_on_data=True` and `bg_on_data=False` respectively.

![Screenshot from 2023-01-10 10-34-27](https://user-images.githubusercontent.com/13835654/211517097-616af5ef-2720-4126-8cad-e66dcbcea7da.png)
![Screenshot from 2023-01-10 10-34-33](https://user-images.githubusercontent.com/13835654/211517100-f50663ee-3d5b-4d2a-93a6-f8ac6c669089.png)